examples/shader/shader_defs.rs

```diff
@@ -23,7 +23,8 @@ pub struct IsRedPlugin;
 impl Plugin for IsRedPlugin {
     fn build(&self, app: &mut App) {
         app.add_plugin(ExtractComponentPlugin::<IsRed>::default());
-        app.sub_app_mut(RenderApp)
+        app.get_sub_app_mut(RenderApp)
+            .unwrap()
```

Comment:
adding an unwrap here sort of defeats the point of using `get_sub_app_mut` over `sub_app_mut`

Reply:
What should I do then?

Reply:
@cart just to be clear: I removed `sub_app_mut` as you suggested to force users handle the error.

Reply:
But panicking API back.